apps/wallet-mobile/src/features/AddWallet/common/CardAboutPhrase/CardAboutPhrase.tsx

```diff
@@ -0,0 +1,84 @@
+import {useTheme} from '@yoroi/theme'
+import React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import {Space} from '../../../../components/Space/Space'
+
+type CardAboutPhraseProps = {
+  items: string[]
+  title?: string
+  background?: boolean
```

Comment:
this is not dubious, API with `background`, a color is expected, if it is fixed, use `hasBackgroundColor` to clearly indicate intention.